apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -47,9 +47,24 @@ const DateDurationResultSchema = z.object({
 });
 
 const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
+  instructor_question_number: z.string().nullable(),
+  manual_rubric_id: IdSchema.nullable(),
+  max_auto_points: z.number().nullable(),
+  max_manual_points: z.number().nullable(),
+  max_points: z.number().nullable(),
   modified_at: z.string(),
   qid: z.string().nullable(),
+  question_id: IdSchema,
   question_number: z.string().nullable(),
+  question_title: z.string().nullable(),
+  row_order: z.number().nullable(),
+  start_new_zone: z.boolean(),
+  zone_best_questions: z.number().nullable(),
+  zone_has_best_questions: z.boolean(),
+  zone_has_max_points: z.boolean(),
+  zone_id: IdSchema.nullable(),
```

Comment:
Is this actually nullable?

Reply:
No, this should never be nullable. Removed `.nullable()` in commit [9ccc082](https://github.com/PrairieLearn/PrairieLearn/pull/9250/commits/9ccc082ef35b88e51bdbf3789d2ef262dbd4fbfc).